00-HTML-CSS-basics/css/style.css

```diff
@@ -0,0 +1,99 @@
+ /*global definition Ex. 2.2.2 */
```

Comment:
I see that there are several styles that looks weird in the view, we will review that later on probably tomorrow. Aside from that. Can you please take some time in order to keep all the code style similar? 

All lines should end with a semicolon at the end of each of them, without a space in between the last char and the semicolon.
All selectors should have a white space before the first curly brace.
Keep 1 white line in between each selector.
Keep all tabs equally as expected.
Review where specific selectors can be grouped in a shorter form. As example: Margins, Paddings.